Sources/UI/Chat/Composer/ComposerView.swift

```diff
@@ -259,8 +259,8 @@ public extension ComposerView {
         styleState = .normal
         previousTextBeforeReset = textView.attributedText
         textView.attributedText = attributedText()
-        uploader?.reset()
-        imageUploaderItems = []
+        uploadManager?.reset()
+        imageUploadingItems = []
```

Comment:
What's the difference between `imageUploadingItems` and `UploadManager.items`?

Reply:
That's the question. I don't remember why, but I'll check now.

Reply:
Fixed/Refactored.